contracts/contracts/RequestManager.sol

```diff
@@ -59,12 +59,18 @@ contract RequestManager is Ownable, LpWhitelist, RestrictedCalls, Pausable {
 
     struct Token {
         uint256 transferLimit;
-        uint96 minLpFee;
+        uint256 price;
         uint32 lpFeePPM;
         uint32 protocolFeePPM;
         uint96 collectedProtocolFees;
     }
 
+    struct Chain {
+        uint256 finalityPeriod;
```

Comment:
this struct could be optimized to fit into one uint256 slot.

Suggestions:

finality period is denominated in seconds and will likely not be higher than 1M. So it would fit well in `uint32`

targetWeightPPM is < 1000000.

and the transferCost could be chosen to fill a uint256 slot but could probably also go down to `uint96`

Reply:
I'm curious, what was the outcome of this discussion?
I imagine both `finalityPeriod` and `targetWeightPPM`could be smaller, e.g. `uint128`, which would reduce the number of slots required by 1.

On the other hand, I'm also ok with this, but would like to know the reasoning of the previous discussion.